Multisig/UI/Transaction/TransactionDetailsViewController/Transaction Detail Cells/RejectionCreatedPiece.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ConfirmationCreatedPiece.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 02.12.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class RejectionCreatedPiece: UINibView {
+
+    @IBOutlet private weak var textLabel: UILabel!
+
+    override func commonInit() {
+        super.commonInit()
+        textLabel.setStyle(.primaryError)
+        // I wish the XIB would allow to set the height constraint to
+        // the file owner, but it doesn't, so we set it in code here
+        NSLayoutConstraint.activate([
```

Comment:
the way how to do it is in the XIB file to have 3 wrappers:

```
View
|---Stack View - pin to the edges of superview
    |---View @ height = 50
        |--- all the internals, or a stack view
```

I even think that it will work if the topmost view will be StackView instead of that wrapper view, because the UINibView can handle the case when the top view in the XIB is stack view

Reply:
Actually I have used the same way used to implement "ConfirmationCreatedPiece"

Now I have updated to just embedded the content in a view which height is 50, without the top two levels Stack view&view, is that good ?

Reply:
if it works, of course!